portfolio/src/main/java/com/google/sps/data/ErrorConsts.java

```diff
@@ -0,0 +1,11 @@
+package com.google.sps.data;
+
+// Stores error messages as constants.
+// Prevents errors due to typos when writing a property name string.
+public class ErrorConsts {
+    public static String EMPTY_VIDEO_ID = "Video id cannot be empty.";
+    public static String VIDEO_NOT_FOUND_IN_DB = "Video not found in database.";
+    public static String UNABLE_TO_RETRIEVE = "Unable to retrieve video.";
+    public static String NO_VIDEO_QUERY_MATCHES = "No such video found.";
+    public static String SECURITY_EXCEPTION = "A security exception occurred.";
+}
```

Comment:
Thanks for adding this!

Reply:
Follow up, I think it would be a good idea to use these in the original code (replace the messages in the original code with ErrorConsts.something).

Reply:
Yeah I'll get to this in a seperate PR